### DISCUSSION TITLE
Delivered Platform Events limit hit due to Log Entry Event Stream component

### DISCUSSION THREAD

**Discussion by cbcruz74** [jongpie/NebulaLogger#512] [Q&A] [answered]
Hi, is there a way to control the Log Entry Event Stream component beyond the Pause button?
I opened the Streaming page during a very busy period (mass data sync), and the 50.000 daily limit was exhausted in minutes.

We also noticed that opening the page again would cause the events to be re-delivered.

**Comment by jongpie** [comment, marked as answer]
Hi @cbcruz74 - perfect timing for your question! I've recently heard from a few other people that have also unintentionally exhausted the daily limit in their orgs, so I am actively working on adding some new features in the next release to provide more control of the Log Entry Event Stream component 🥳 I'll also try to address the events being re-delivered when reopening the page

My changes for this will be in pull request #507, which will be released as `v4.11.0`. I'm hoping to have it released by the end of this week or early next week.

**Reply by jongpie** [reply]
@cbcruz74 just wanted to follow up on this - earlier today, I released [`v4.11.0`](https://github.com/jongpie/NebulaLogger/releases/tag/v4.11.0), which includes some enhancements for the component to limit how many platform events it streams.

**Reply by cbcruz74** [reply]
Thanks, @jongpie . We'll take a look and test it.
